README.md

```diff
@@ -1,39 +1,117 @@
+
+### Project Objectives <a name="project-objectives"></a>
+
+- [ ] Write plain HTML markup with minimum styling (just to make the layout) using the wireframe below as a guide:
+
+<h3 align="center">Wireframe Provided</h3>
+<p align="center">
+  <img width="500" src="screenshot/leaderboard_wireframe.png">
+</P>
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML)
+- [JavaScript/ES6](https://262.ecma-international.org/6.0/)
+- [webpack 5](https://webpack.js.org/)
+- [npm](https://www.npmjs.com/)
+- [Git](https://git-scm.com/)
+- [GitHub](https://github.com)
+- [Stylelinter](https://stylelint.io/)
+- [ESLint](https://eslint.org/)
+- [Webhint](https://webhint.io/)
+- [Lighthouse](https://ethcar.github.io/lighthouse/)
+- [SASS](https://sass-lang.com/)
+
+### Key Features <a name="key-features"></a>
+
+- [x] Use ES6 modules, with import and export.
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> [Live Demo Link](https://EngruugbeJimmy.github.io/leaderboard/)
+
+<p align="center">
+  <img width="500" src="screenshot/page_demo.png">
+</P>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+- [Optional] Install git bash to your machine to enable you to clone this repo.
+- install Visual Studio to be able to host a local live version.
+- Install a browser to view the local live version.
+
+To get a local copy up and running follow these simple example steps.
+### Setup <a name="setup"></a>
+
+- Open your GitHub account the repository's [link](https://github.com/EngruugbeJimmy/leaderboard)
+
+### Prerequisites <a name="prerequisites"></a>
+
+- Internet connection
+- A github account
+### Install <a name="install"></a>
+
+- copy the repo's link and clone it by writing `git clone https://github.com/EngruugbeJimmy/leaderboard.git` on your git bash terminal.
+- `npm install` to install the dependencies
+
+### Run tests <a name="run-tests"></a>
+
+- You can check for errors by running linter tests found in the github flows.
+
+### Deployment <a name="deployment"></a>
+
+- run `npm run start` to run it locally.
+
+## Authors <a name="authors"></a>
+
+👤 **EngruugbeJimmy**
+
+- [GitHub](https://github.com/EngruugbeJimmy)
+- [Twitter](https://twitter.com/abbaJdev)
+- [LinkedIn](https://www.linkedin.com/in/abah-james-ugbede-356982159/)
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
```

Comment:
Kindly add the section `Future Features`, including 1-3 potential features to be added. All the main headers from this [README template](https://github.com/microverseinc/readme-template) should be present in your repo. The only optional section is FAQ.